### ISSUE TITLE
Question about the BEV projection of the extracted feature

### ISSUE THREAD

**Issue by yuan-zm** [valeoai/BEVContrast#2]
Thank you for this excellent work.

One thing make me confused.

https://github.com/valeoai/BEVContrast/blob/54497c9ecfc71623d18f7974988a0c2d14b93b99/bevlab/models.py#L54

1. Why project features onto a BEV image and then perform rotation and translation?

3. Would it be more convenient to rotate and translate the point cloud and then project the feature into the BEV image? It also does not need to resample the feature by F.affine_grid and F.grad_sample later.
https://github.com/valeoai/BEVContrast/blob/54497c9ecfc71623d18f7974988a0c2d14b93b99/bevlab/models.py#L98
https://github.com/valeoai/BEVContrast/blob/54497c9ecfc71623d18f7974988a0c2d14b93b99/bevlab/models.py#L100

**Comment by CSautier** [comment]
Hi, thank you for your interest in the work.
Your question is very relevant and was as much of a surprise to us as it seems to be to you. I invite you to look at the table 6 in the main article addressing exactly this point, but in summary we tried it as well, and it worked slightly less well.
My personnal intuition is that the bilinear interpolation itself acts as a spatial regularizer, but I don't have any other proper experiment to back this claim.

A second argument for keeping the affine transform in 2D is because it can then use the same code for BEV architectures, such as the ones we used for object detection.
